### PR TITLE
Resumes

### PR DESCRIPTION
**Pull request by g4s8**
zerocracy/farm#800 - resumes schema, tests, xsl

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#334](https://github.com/zerocracy/datum/issues/334) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#334](https://github.com/zerocracy/datum/issues/334) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/850944422); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer

File: xml/pmo/resumes/-duplicates.xml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <submitted>2018-01-01T00:00:00</submitted>
+    <examiner>g4s8</examiner>
+    <text>I'm great</text>
+    <personality>INTJ-A</personality>
+    <stackoverflow>187141</stackoverflow>
+    <telegram>yegor256</telegram>
+  </resume>
+  <resume login="yegor256">
+    <submitted>2018-01-01T00:00:00</submitted>
+    <examiner>g4s8</examiner>
```
Comment (carlosmiranda) [inline xml/pmo/resumes/-duplicates.xml]: @g4s8 since we're enforcing uniqueness of the login attribute, let's make the rest of data different.

**Comment by g4s8** [comment]
@carlosmiranda fixed in f921594, please look again

**Comment by carlosmiranda** [comment]
@yegor256 good to merge

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#334](https://github.com/zerocracy/datum/issues/334) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda Thanks.

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**